cypress/integration/inventory/create-edit-delete-material-types.spec.js

```diff
@@ -3,7 +3,8 @@ import permissions from '../../support/dictionary/permissions';
 import SettingsMenu from '../../support/fragments/settingsMenu';
 import getRandomPostfix from '../../support/utils/stringTools';
 import InventorySettings from '../../support/fragments/inventory/inventorySettings';
-import MaterialTypesSettings from '../../support/fragments/inventory/materialType/materialTypesSettings';
+import MaterialTypes from '../../support/fragments/settings/inventory/materialTypes';
+import NewMaterialType from '../../support/fragments/settings/inventory/newMaterialType';
 import users from '../../support/fragments/users/users';
```

Comment:
Please import the fragment starting from the UpperCase letter

Reply:
Done